anime_downloader/sites/animeonline.py

```diff
@@ -16,6 +16,10 @@ def search(cls, query):
                 SearchResult(
                     title = i['title'],
                     url = i['url'],
+                    meta_info = {

```

Comment:
What is this meta info exactly?

Reply:
Metadata which doesn't show up in search.